platform/ios/Makefile

```diff
@@ -4,6 +4,10 @@ project_path = PolyPodApp/PolyPod.xcodeproj
 scheme_name = PolyPod
 export_options_plist = exportOptions.plist
 destination = 15.2
+appstore_api_key_id = $(POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID)
+appstore_api_issuer_id = $(POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID)
+provisioning_profile_dir := ~/Library/MobileDevice/Provisioning\ Profiles
```

Comment:
Nit: Variables could be defined also for other paths like `~/Library/Keychains/build.keychain`.

Reply:
Yeah, I considered that, but didn't want to rewrite the whole thing :D This one I just moved to a variable to get rid of the ridiculously long lines and slight duplication we had down there.